### ISSUE TITLE
IP Forwarding Error on Join

### ISSUE THREAD

**Issue by nicholasveronico** [GeyserMC/Floodgate#272]
When a player joins, they are sometimes greeted with the kick message: "Kicked whilst connecting to BedrockHub: If you wish to use IP forwarding, please enable it in your BungeeCord config as well!" The issue is not a misconfiguration, as all it takes is a player to rejoin and they are let into the server, "BedrockHub".

This issue started on February 2, when the Mojang API change patch was implemented. This issue only happens when `send-floodgate-data` is set to true.

To reproduce: I tried this on a Waterfall proxy, with the only plugins being floodgate-bungee and Geyser-Bungeecord. The backend server called BedrockHub is a paper 1.8.8 server with the only plugin being floodgate. The config option `send-floodgate-data` is set to true on the proxy.

This is the log showing that rejoining lets the player in: https://mcpaste.io/9f03d044f06f3487

If you need any additional information, please let me know. I have attached my config for Geyser Bungeecord, Floodgate Bungee and Floodgate Spigot.
[Configs.zip] (https://github.com/GeyserMC/Floodgate/files/8052522/Configs.zip)

Thank you.

**Comment by Kas-tle** [comment]
Posting conversation about this issue from Geyser Discord in case it is relevant:

```
Camotoy — Today at 11:39 AM
Just tested and I can replicate the issue using Spigot 1.8.8.
[11:39 AM]
So likely something about using the older version, maybe. (edited)
[11:39 AM]
I remember someone having the issue on 1.16.5.
[11:39 AM]
On 1.17 it's fine.
 
ItzFlip — Today at 11:39 AM
Is there anything that could be done about that?
 
Camotoy — Today at 11:40 AM
I'm sure there is.
 
ItzFlip — Today at 11:41 AM
alright. Please let me know if you need me to test anything
```

**Comment by Tim203** [comment]
Could you try using `dev/2.1.1`?
I think that I might know what's causing this and that should be fixed in `dev/2.1.1`

**Comment by juzijun233** [comment]
> Could you try using `dev/2.1.1`? I think that I might know what's causing this and that should be fixed in `dev/2.1.1`

The firewall problem under dev2.1.1 is solved

**Comment by Tim203** [comment]
I expect that this issue has been solved like I said above.
Feel free to comment if you're still experiencing this.
